backend.rb

```diff
@@ -57,37 +46,19 @@ def build_params_for_authy(*params_to_copy)
   DOCUMENTATION
 end
 
-post "/registration/start" do
-  param :country_code, Integer, required: true
-  param :phone_number, String, required: true
-  param :email, String, required: true
-  param :via, String, required: true, in: ["sms", "call"]
-
-  params_for_authy = build_params_for_authy(:country_code, :phone_number, :email, :via)
-
-  response = RestClient.post(build_url("/protected/json/registrations/start"), params_for_authy)
-
-  if response.code == 200
-    respond_with status: StatusOK
-  end
-
-  respond_with status: StatusFailed
-end
+post "/registration" do
+  param :authy_id, Integer, required: true
 
-post "/registration/complete" do
-  param :country_code, Integer, required: true
-  param :phone_number, String, required: true
-  param :pin, String, required: true
+  params_for_authy = build_params_for_authy(:authy_id)
 
-  params_for_authy = build_params_for_authy(:country_code, :phone_number, :pin)
+  response = RestClient.post(build_url("/protected/json/sdk/registrations"), params_for_authy) {|r| r }
 
-  response = RestClient.post(build_url("/protected/json/registrations/confirm"), params_for_authy)
+  authy_response = JSON.parse(response.body)
```

Comment:
@dcu thanks, fix included in PR: https://github.com/authy/authy-sdk-backend/pull/2